app/js/header5.js

```diff
@@ -0,0 +1,44 @@
+var touches = document.querySelectorAll('.js-touch__btn');
+var left = document.querySelector('.js-arrow__left');
+var right = document.querySelector('.js-arrow__right');
+var number = 0;
+
+for(var i = 0; i < touches.length; i++) {
+    touches[i].addEventListener('click', Click(i));
+}
+
+function Click(index) {
+    return function (e) {
+        number = index;
+        clearClasses();
+        this.classList.add('fifth__touch--active');
```

Comment:
Please, move `'fifth__touch--active'` to variable and use variable everywhere. Like:
`var active = 'fifth__touch--active'`

Reply:
Corrected